src/lib.rs

```diff
@@ -320,15 +321,29 @@ impl Builder {
 		self
 	}
 
-	/// Builds a [`Node`] instance according to the options previously configured.
-	pub fn build(&self) -> Arc<Node> {
-		let config = Arc::new(self.config.clone());
+	/// Builds a [`Node`] instance with a [`SqliteStore`] backend and according to the options
+	/// previously configured.
+	pub fn build(&self) -> Arc<Node<SqliteStore>> {
+		let storage_dir = self.config.storage_dir_path.clone();
+		fs::create_dir_all(storage_dir.clone()).expect("Failed to create LDK data directory");
+		let kv_store = Arc::new(SqliteStore::new(storage_dir.into()).into());
+		self.build_with_store(kv_store)
+	}
 
-		let ldk_data_dir = format!("{}/ldk", config.storage_dir_path);
-		fs::create_dir_all(ldk_data_dir.clone()).expect("Failed to create LDK data directory");
+	/// Builds a [`Node`] instance with a [`FilesystemStore`] backend and according to the options
+	/// previously configured.
+	pub fn build_with_fs_store(&self) -> Arc<Node<FilesystemStore>> {
```

Comment:
can't we do similar to set_esplora_server_url ? 
and use same builder pattern for this.

Reply:
moved to another PR